Formula/maxwell.rb

```diff
@@ -0,0 +1,39 @@
+class Maxwell < Formula
+  desc "Maxwell's daemon, a mysql-to-json kafka producer"
+  homepage "http://maxwells-daemon.io/"
+  url "https://github.com/zendesk/maxwell/releases/download/v1.13.1/maxwell-1.13.1.tar.gz"
+  sha256 "3ab7489d47f2ec04677122c8786cf573572b9bbb511b184802d2dcfe82e83781"
+
+  bottle do
+    cellar :any
+    sha256 "3ab7489d47f2ec04677122c8786cf573572b9bbb511b184802d2dcfe82e83781" => :high_sierra
+    sha256 "3ab7489d47f2ec04677122c8786cf573572b9bbb511b184802d2dcfe82e83781" => :sierra
+    sha256 "3ab7489d47f2ec04677122c8786cf573572b9bbb511b184802d2dcfe82e83781" => :el_capitan
+  end
+
+  depends_on "maven" => :build
```

Comment:
You're not building anything, so this seems unlikely.

Reply:
We can build it using makefile (which in turn uses maven) or simply use the shell script to run.  Since this takes the latter option, I removed maven.